### PR TITLE
KOGITO-3007 Improve skip parameters in release

### PR DESCRIPTION
**Pull request by radtriste**
Depends on https://github.com/kiegroup/kogito-pipelines/pull/40

### CODE REVIEW

File: Jenkinsfile.release [hunk 1]
```diff
@@ -36,9 +36,12 @@ pipeline {
         string(name: 'EXAMPLES_URI', defaultValue: '', description: 'Override default. Git uri to the kogito-examples repository to use for tests.')
         string(name: 'EXAMPLES_REF', defaultValue: '', description: 'Override default. Git reference (branch/tag) to the kogito-examples repository to use for tests.')
 
-        booleanParam(name: 'SKIP_RUNTIMES', defaultValue: false, description: 'To skip Runtimes. If skipped, please provide `ARTIFACTS_REPOSITORY`')
-        booleanParam(name: 'SKIP_IMAGES', defaultValue: false, description: 'To skip Images Deployment')
-        booleanParam(name: 'SKIP_OPERATOR', defaultValue: false, description: 'To skip Operator Deployment')
+        booleanParam(name: 'SKIP_RUNTIMES_DEPLOY', defaultValue: false, description: 'To skip Runtimes Test & Deployment. If skipped, please provide `ARTIFACTS_REPOSITORY`')
+        booleanParam(name: 'SKIP_RUNTIMES_PROMOTE', defaultValue: false, description: 'To skip Runtimes Promote only. Automatically skipped if SKIP_RUNTIMES_DEPLOY is skipped.')
```
Comment (Kevin-Mok) [inline Jenkinsfile.release]:
```suggestion
        booleanParam(name: 'SKIP_RUNTIMES_PROMOTE', defaultValue: false, description: 'To skip Runtimes Promote only. Automatically skipped if SKIP_RUNTIMES_DEPLOY is true.')
```

I would change this wording here for clarity.

File: Jenkinsfile.release [hunk 2]
```diff
@@ -36,9 +36,12 @@ pipeline {
...
+        booleanParam(name: 'SKIP_IMAGES_DEPLOY', defaultValue: false, description: 'To skip Images Test & Deployment.')
+        booleanParam(name: 'SKIP_IMAGES_PROMOTE', defaultValue: false, description: 'To skip Images Promote only. Automatically skipped if SKIP_IMAGES_DEPLOY is skipped.')
```
Comment (Kevin-Mok) [inline Jenkinsfile.release]:
```suggestion
        booleanParam(name: 'SKIP_IMAGES_PROMOTE', defaultValue: false, description: 'To skip Images Promote only. Automatically skipped if SKIP_IMAGES_DEPLOY is true')
```

File: Jenkinsfile.release [hunk 3]
```diff
@@ -36,9 +36,12 @@ pipeline {
...
+        booleanParam(name: 'SKIP_OPERATOR_DEPLOY', defaultValue: false, description: 'To skip Operator Test & Deployment.')
+        booleanParam(name: 'SKIP_OPERATOR_PROMOTE', defaultValue: false, description: 'To skip Operator Promote only. Automatically skipped if SKIP_OPERATOR_DEPLOY is skipped.')
```
Comment (Kevin-Mok) [inline Jenkinsfile.release]:
```suggestion
        booleanParam(name: 'SKIP_OPERATOR_PROMOTE', defaultValue: false, description: 'To skip Operator Promote only. Automatically skipped if SKIP_OPERATOR_DEPLOY is true.')
```

**Review by Kevin-Mok — APPROVED** [review]

**Review by Ginxo — APPROVED** [review]
LGTM

**Pull request merged**